account_product_fiscal_classification/tests/tests.py

```diff
@@ -138,3 +144,12 @@ def test_06_unlink_fiscal_classification(self):
             [self.classification_1_id])
         with self.assertRaises(ValidationError):
             classification.unlink()
+
+    def test_07_classification_generate_from_template(self):
```

Comment:
@pedrobaeza @danimaribeiro this one is driving me crazy. I tried both suggestions and tests keep not running:
https://github.com/akretion/account-fiscal-rule/commit/f61bd3ae037ba1e3b1daca2e2974c85b278889d5
https://github.com/akretion/account-fiscal-rule/commit/62cbb6cc04e11b8c64d8614b19d0ee11e8a67412

I'm testing this on my 2Gb i3 slow disk super laptop then now... (got a laptop crash)